test/tail.js

```diff
@@ -1,19 +1,22 @@
 'use strict';
 
-const S = require ('..');
+const S = require ('./internal/sanctuary');
 
 const eq = require ('./internal/eq');
+const {Nil, Cons} = require ('./internal/List');
 
 
 test ('tail', () => {
 
   eq (typeof S.tail) ('function');
   eq (S.tail.length) (1);
-  eq (S.show (S.tail)) ('tail :: Array a -> Maybe (Array a)');
+  eq (S.show (S.tail)) ('tail :: (Foldable f, Applicative f) => f a -> Maybe (f a)');
 
   eq (S.tail ([])) (S.Nothing);
   eq (S.tail (['foo'])) (S.Just ([]));
   eq (S.tail (['foo', 'bar'])) (S.Just (['bar']));
   eq (S.tail (['foo', 'bar', 'baz'])) (S.Just (['bar', 'baz']));
 
+  eq (S.tail (Cons ('baz') (Cons ('foo') (Nil)))) (S.Just (Cons ('foo') (Nil)));
+  eq (S.tail (Nil)) (S.Nothing);
 });
```

Comment:
Please insert a blank line after the last assertion.

Reply:
This file has been updated.